test/torch/tensors/test_native.py

```diff
@@ -191,6 +193,8 @@ def forward(self, x):
     ## Forward on the remote model
     pred = model_net(tensor_remote)
 
+    model_net.get()
```

Comment:
Can we add some assertions in this test?

Reply:
done!